gub/3/new.py

```diff
@@ -1,4 +1,4 @@
-# https://docs.python.org/3.0/whatsnew/3.0.html
+# http://docs.python.org/3.0/whatsnew/3.0.html
```

Comment:
err, I think you meant to rebase it into e19811e84ac7720c29b22788e8655cce85073361 ?  The whole idea is to avoid unnecessary changes to the code.  Currently, e19811 makes a change, then this one (125f04b57) changes it back.  Please alter e19811.

The good news is that once you've changed e19811, modern git will recognize that you've already made the change in this commit, so it will fix 125f04b57 without any extra work on your part.